external/unity/RoslynBuilder/RoslynBuilder/BuildCscStep.cs

```diff
@@ -10,20 +10,34 @@ class BuildCscStep : IBuildStep
 		public void Execute()
 		{
 			BuildCsc("netcoreapp2.0", "win-x64", KnownPaths.CscWindowsBinariesDirectory);
+			BuildVBCSCompiler("netcoreapp2.0", "win-x64", KnownPaths.CscWindowsBinariesDirectory);
 			BuildCsc("netcoreapp2.0", "osx-x64", KnownPaths.CscMacBinariesDirectory);
+			BuildVBCSCompiler("netcoreapp2.0", "osx-x64", KnownPaths.CscMacBinariesDirectory);
 			BuildCsc("netcoreapp2.0", "linux-x64", KnownPaths.CscLinuxBinariesDirectory);
+			BuildVBCSCompiler("netcoreapp2.0", "linux-x64", KnownPaths.CscLinuxBinariesDirectory);
 			BuildCsc("net46", null, KnownPaths.CscNet46Directory);
+			BuildVBCSCompiler("net46", null, KnownPaths.CscNet46Directory);
```

Comment:
Perhaps instead of having 2 of these functions just have 1 function called "BuildRoslynComponents" which calls BuildProject twice: once for Csc and once for VBCS?

Reply:
Sure, I wasn't sure how you wanted to organize the code reuse